haproxy/README.md

```diff
@@ -1,5 +1,6 @@
 # Haproxy Integration
-{{< img src="integrations/haproxy/haproxydash.png" alt="HAProxy default dashboard" responsive="true" popup="true">}}
+
+{{< img src="https://raw.githubusercontent.com/DataDog/integrations-core/master/haproxy/images/haproxy-dash.png" alt="HAProxy default dashboard" responsive="true" popup="true">}}
```

Comment:
This won't work, partial are made to pull the image from the documentation image folder. 

If you want to display an image on this README.md you should use the markdown layout: `![my_image](link/to/my/image)` or keep the partial as it is.

Reply:
I don't know if this is a big deal, but I permalink to the file through its commit rather than `master`.

Reply:
Ok, thanks for the info. Changed it to a markdown link with a reference to the commit instead of master.